src/scripts/main.js

```diff
@@ -1 +1,78 @@
 'use strict';
+
+function notification(value) {
+  const div = document.createElement('div');
+
+  div.setAttribute('data-qa', 'notification');
+  div.classList = value[0];
+  div.innerText = value[1];
+  document.body.appendChild(div);
+}
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', (e) => {
+    const value = ['success', 'First promise was resolved'];
+
+    resolve(value);
+
+    e.preventDefault();
+  });
+
+  setTimeout(() => {
+    const value = ['error', 'First promise was rejected'];
+
+    reject(value);
+  }, 3000);
+});
+
+firstPromise
+  .then((value) => {
+    notification(value);
+  })
+
+  .catch((value) => {
+    notification(value);
+  });
+
+const secondPromise = new Promise((resolve) => {
+  const value = ['success', 'Second promise was resolved'];
+
+  document.addEventListener('click', (e) => {
+    resolve(value);
+  });
+
+  document.addEventListener('contextmenu', (e) => {
+    resolve(value);
+  });
```

Comment:
The 'contextmenu' event listener resolves the second promise immediately upon any right-click. This might not be the intended behavior if the promise should only resolve under specific conditions. Consider adding conditions to verify the event target or other criteria before resolving.